app/assets/javascripts/components/markdown-editor.js

```diff
@@ -0,0 +1 @@
+//= require markdown-toolbar-element/dist/index.umd.js
```

Comment:
Should this file be marked as "global"? https://github.com/alphagov/content-publisher/blob/d272ddc0dd00d8469c8a58e787624f874bf3b0aa/app/assets/javascripts/components/markdown-editor.js#L1